start.sh

```diff
@@ -53,11 +53,13 @@ for CONTAINER in $CONTAINERS; do
 	docker exec -ti $CONTAINER /bin/bash -c 'until [[ -f .htaccess ]]; do sleep 1; done'
 	docker exec -ti $CONTAINER /bin/bash -c 'wp --allow-root core is-installed 2>/dev/null'
 	IS_INSTALLED=$?
+
+	docker exec -ti $CONTAINER /bin/bash -c "usermod -u ${USER_ID} www-data"
+	docker exec -ti $CONTAINER /bin/bash -c "groupmod -o -g ${GROUP_ID} www-data"
+	docker container restart $CONTAINER
+
```

Comment:
What was the reason for moving / editing this? I am noticing an increase in boot time for Docker for every container I intent to start (5 to 10 sec. per container or so), I guess due to the restarts.

Reply:
Because otherwise the rights are not matching and I cannot upload/modify any files on the web-instance using Docker with the WSL integration.
These seem to have to be set every time the instances are started.
In my WSL environment this takes less than a second though 🤔 
The restart seems to be required; to make sure the service is running as the expected user.

Reply:
I will investigate further and come back with more information.